### ISSUE TITLE
Normal maps ignored when using Max procedural maps

### ISSUE THREAD

**Issue by juancarlosgzrz** [appleseedhq/appleseed-max#81]

**Comment by dictoon** [comment]
@juancarlosgzrz Can you confirm that this issue is still relevant?

**Comment by juancarlosgzrz** [comment]
Yes, it is, they behave differently when using Max procedurals. I'll atach renders.

**Comment by dictoon** [comment]
Confirmed to also happen with bump mapping.

**Comment by usakhelo** [comment]
@juancarlosgzrz, this is fixed in master branch, please let us know how it works when you have a chance to check.

**Comment by dictoon** [comment]
Bump mapping is fixed in master, but normal mapping is using an entirely different mechanism.
